Bottom: 056b8bfd424c0057063f61bff98567c021510bde
Top:    0f29fc79a2e44b45051dc6d32b885f356ac262aa
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-10-24 15:40:19 -0700

Refresh of rs-iomap

---

diff --git a/src/librdmacm.map b/src/librdmacm.map
index 5c317a3..d5ef736 100644
--- a/src/librdmacm.map
+++ b/src/librdmacm.map
@@ -63,5 +63,8 @@ RDMACM_1.0 {
 		rselect;
 		rdma_get_src_port;
 		rdma_get_dst_port;
+		riomap;
+		riounmap;
+		riowrite;
 	local: *;
 };
